app/src/main/java/website/bloop/app/BloopApplication.java

```diff
@@ -0,0 +1,36 @@
+package website.bloop.app;
+
+import android.app.Application;
+
+import com.google.android.gms.common.api.GoogleApiClient;
+
+/**
+ *
+ */
+public class BloopApplication extends Application {
+    private static BloopApplication mInstance = null;
+
+    private GoogleApiClient mGoogleApiClient;
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+        mInstance = this;
+    }
+
+    public static BloopApplication getInstance(){
+        if(mInstance == null)
```

Comment:
small nit: keep the starting bracket on the same line as the if, and have a space after the `if` and before the `(`.